test/integration_test/health_monitor_test.go

```diff
@@ -25,6 +26,8 @@ const (
 	nodeOfflineTimeout = 295 * time.Second
 )
 
+var dash = aetosutil.Get()
```

Comment:
We are already getting the `dash` object in `tests/common.go` you can reuse it:

https://github.com/libopenstorage/stork/blob/e07b12b5153c622444f3d7a05580d42b7de4af9a/test/integration_test/common_test.go#L333